packages/transaction-controller/src/types.ts

```diff
@@ -490,6 +490,39 @@ export type TransactionMeta = {
   };
 };
 
+export type TransactionBatchMeta = {
+  /**
+   * Network code as per EIP-155 for this transaction.
+   */
+  chainId: Hex;
+
+  /**
+   * ID of the associated transaction batch.
+   */
+  id: string;
+
+  /**
+   * Data for any nested transactions.
```

Comment:
Technically not "nested" as that is EIP-7702 only.

Reply:
Done